estate/models/estate_property_offer.py

```diff
@@ -0,0 +1,92 @@
+from datetime import timedelta
+
+from odoo import fields, models, api
+from odoo.exceptions import UserError, ValidationError
+
+
+class EstatePropertyOffer(models.Model):
+    _name = "estate.property.offer"
+    _description = "Property Offers for Real Estate App"
+    _order = "price desc"
+
+    price = fields.Float(string="Price")
+    status = fields.Selection(
+        selection=[
+            ("accepted", "Accepted"),
+            ("refused", "Refused"),
+        ],
+        copy=False,
+        string="Status",
+    )
+    partner_id = fields.Many2one("res.partner", string="Partner", required=True)
+    property_id = fields.Many2one("estate.property", string="Property", required=True)
+    validity = fields.Integer(default=7)
+    date_deadline = fields.Date(
+        compute="_compute_date_deadline", inverse="_inverse_date_deadline"
+    )
+    property_type_id = fields.Many2one(
+        related="property_id.property_type_id", store=True
+    )
+
+    _sql_constraints = [
+        (
+            "check_offer_price",
+            "CHECK(price > 0)",
+            "Offer Price for the property should be strictly positive",
+        )
+    ]
+
+    @api.depends("validity")
+    def _compute_date_deadline(self):
+        for record in self:
+            create_date = (
+                record.create_date.date() if record.create_date else fields.Date.today()
+            )
```

Comment:
```suggestion
            create_date = record.create_date.date() if record.create_date else fields.Date.today()
```
Unnecessary brackets